### DISCUSSION TITLE
Compilation

### DISCUSSION THREAD

**Discussion by Lauorez** [LeagueOfPoro/CapsuleFarmerEvolved#107] [Q&A]
My question is short and simple: how was it compiled to an EXE?

**Comment by XaaRii** [comment]
using pyinstaller.
more info here: https://github.com/LeagueOfPoro/CapsuleFarmerEvolved/wiki/Advanced-Installation-for-Windows

**Comment by WatchingCat** [comment]
umm, about the step 4:Run setup/windows.bat (CMD/PowerShell compatible)
how can I find this file(setup/windows.bat)? where is it come from?

**Reply by XaaRii** [reply]
not sure as I always ran from source, but by best guess is that after installing pyinstaller and following the steps something would generate it..?
